packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -92,66 +94,69 @@ Cons:
 
 ### API
 
-#### Icon column element:
-
-_Component Name_
-
--   `nimble-table-column-icon`
-
-_Props/Attrs_
-
--   `field-name`: string
--   `key-type`: 'string' | 'number' | 'boolean'
-
-_Content_
-
--   column title (icon)
--   1 or more `nimble-mapping-icon` or `nimble-mapping-spinner` elements
-
-#### General mapping column element:
+#### Enum column element:
 
 _Component Name_
 
--   `nimble-table-column-enum-text`
+-   `nimble-table-column-enum`
 
 _Props/Attrs_
 
 -   `field-name`: string
 -   `key-type`: 'string' | 'number' | 'boolean'
--   `fractional-width`: number (defaults to 1)
--   `min-pixel-width`: number (defaults to minimum supported by table)
+-   `fixed-width-mode`: enum - `flexibleWidth` (default) | `iconWidth` - When set to `iconWidth`, the column will have a fixed width that makes the column the appropriate width to render only a single icon in the cell. This should only be set when the header contains a single icon (no text) and none of the child mapping elements will result in text being rendered in a cell.
 
 _Content_
 
--   column title (text or icon)
--   1 or more `nimble-mapping-text` elements
+-   column title (icon and/or text)
+-   1 or more `nimble-mapping-icon`, `nimble-mapping-spinner`, `nimble-mapping-text`, or `nimble-mapping-empty` elements
 
 #### Mapping element (icon):
 
+The icon mapping element will support displaying an icon, icon with text, or only text in a cell. If only text is displayed in a cell, space will be reserved for an icon so that the text associated with all icon mapping elements is aligned. A group row associated with an icon mapping element will always display the icon, if specified, and text.
```

Comment:
It took me a minute to wrap my head around the distinction between an icon mapping with no icon and a text mapping. I think it makes sense, but we'll probably want to write usage guidance in storybook that explains which combinations of mappings are recommended. I'm guessing we wouldn't suggest mixing text mappings with text-only-icon mappings since they'd end up out of alignment.

No action needed in this PR, just a reminder for when you get to the implementation PR(s).